INSTALL.rst

```diff
@@ -219,9 +245,16 @@ In order to build the documentation you must have additionally
 ``pandoc`` installed. Use your system’s package manager to do this
 (e.g. ``sudo apt-get install pandoc``).
 
-You can build the entire documentation simply by using sphinx from
-``setup.py``:
+You can build the entire documentation simply by using the ``generate_docs.sh``
+script:
 
 .. code:: sh
 
-   python3 setup.py build_ext --inplace docs
+   bash docs/generate_docs.sh DIRECTORY PYTHON_EXECUTABLE [use_virtual_env]
```

Comment:
Please make it:
```
bash docs/generate_docs.sh DIRECTORY [PYTHON_EXECUTABLE] [use_virtual_env]
```
Hardly ever is `python3`, i.e. the one in the `PATH` not the one wants.